result.go

```diff
@@ -48,23 +48,12 @@ type RequestFilterForRunResults struct {
 // SendableResult represents a Test Case result
 // that can be created or updated via the api
 type SendableResult struct {
-	StatusID     int                `json:"status_id,omitempty"`
-	Comment      string             `json:"comment,omitempty"`
-	Version      string             `json:"version,omitempty"`
-	Elapsed      string             `json:"elapsed,omitempty"`
-	Defects      string             `json:"defects,omitempty"`
-	AssignedToID int                `json:"assignedto_id,omitempty"`
-	Checkbox     bool               `json:"custom_checkbox,omitempty"`
```

Comment:
Those fields are presented as "Custom fields" on the documentation: http://docs.gurock.com/testrail-api2/reference-results